### ISSUE TITLE
[Feature Request]: Accessibility option to allow Ultimine keybind to work as a Toggle alongside current Hold behaviour

### ISSUE THREAD

**Issue by osixjoh** [FTBTeam/FTB-Mods-Issues#1093]
### Mod

FTB Ultimine

### Description

For accessibility purposes, the keybind that players need to hold down to use Ultimine should have a configuration option to switch between behaving as a 'toggle' and a 'held' keybind. The toggle behaviour would allow a player could to the keybind once to 'enable' Ultimine, and press it again to 'disable' Ultimine, instead of this behaviour being linked to holding down and releasing the keybind respectively.

### Describe the solution you'd like

Add a configuration option to allow players to change Ultimine's behaviour from Hold to Toggle, like the accessibility options for Sneak and Sprint.
